transcriptLoader.py

```diff
@@ -0,0 +1,116 @@
+import glob
+from datetime import datetime
+import pandas as pd
+
+
+def processSrtFile(srtFile):
+    """
+    Process an SRT file and extract the transcript data.
+
+    Args:
+        srtFile (str or list): The path to the SRT file or a list of paths.
+
+    Returns:
+        pandas.DataFrame: A DataFrame containing the extracted transcript data with columns 'Line', 'Start', and 'End'.
+    """
+    if type(srtFile) == list:
+        srtFile = srtFile[0]
+
+    with open(srtFile, "r") as f:
+        lines = f.readlines()
+
+    transcript = []
+
+    sentence = ""
+    startTime, endTime = "", ""
+
+    for line in lines:
+        line = line.strip()
+        if line.isdigit():
+            continue
+        elif "-->" in line:
+            startTime, endTime = line.split("-->")
+            startTime = datetime.strptime(startTime.strip(), "%H:%M:%S,%f")  # .time()
+            endTime = datetime.strptime(endTime.strip(), "%H:%M:%S,%f")  # .time()
+        elif line:
+            sentence += " " + line
+        else:
+            transcript.append(
+                {"Line": sentence.strip(), "Start": startTime, "End": endTime}
+            )
+            sentence = ""
+
+    return pd.DataFrame(transcript)
```

Comment:
I would make this a variable and then return the variable that hold the `df`. good for debugging purposes. also log what is returned as debug

```
transcript_df = pd.DataFrame(transcript)
return transcript_df
```

Reply:
Same applies to L80